src/app/core/entity/schema-datatypes/datatype-date-only.spec.ts

```diff
@@ -2,9 +2,23 @@ import { dateOnlyEntitySchemaDatatype } from "./datatype-date-only";
 
 describe("Schema data type:Date", () => {
   it("should not fail on null values", () => {
-    const nullDateRes = dateOnlyEntitySchemaDatatype.transformToDatabaseFormat(
-      null
-    );
+    const nullDateRes =
+      dateOnlyEntitySchemaDatatype.transformToDatabaseFormat(null);
     expect(nullDateRes).toBeUndefined();
   });
+
+  it("should correctly transform values", () => {
+    const date = new Date(2022, 0, 1);
+
+    const dbFormat =
+      dateOnlyEntitySchemaDatatype.transformToDatabaseFormat(date);
+    expect(dbFormat).toBe("2022-01-01");
+
+    const objFormat: Date =
```

Comment:
maybe we should consider writing a test for different timezones? https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config

Reply:
Unfortunately, I did not find a way of achieving the same with `jasmine`.

Reply:
But if you set your computer to a different timezone and execute the test which I wrote, it will test for this bug/issue.